internal/controller/pipelinerollout_controller_test.go

```diff
@@ -152,6 +157,24 @@ var _ = Describe("PipelineRollout Controller", func() {
 			currentPipelineRollout := &apiv1.PipelineRollout{}
 			Expect(k8sClient.Get(ctx, resourceLookupKey, currentPipelineRollout)).ToNot(HaveOccurred())
 
+			var lastTransitionTime time.Time
```

Comment:
Do you mind adding a test for checking the annotation is there after the first creation?